tests/js/spec/views/organizationIntegrations/integrationListDirectory.spec.jsx

```diff
@@ -0,0 +1,260 @@
+              owner: {id: 1, slug: 'sentry'},
+              redirectUrl: null,
+              schema: {},
+              scopes: [],
+              slug: 'clickup',
+              status: 'published',
+              uuid: '5d547ecb-7eb8-4ed2-853b-40256177d526',
+              verifyInstall: false,
+              webhookUrl: 'http://localhost:7000',
+            },
+          ],
+        ],
+        [
+          `/organizations/${org.slug}/sentry-app-installations/`,
+          [
+            {
+              app: {
+                slug: 'my-headband-washer-289499',
+                uuid: 'a806ab10-9608-4a4f-8dd9-ca6d6c09f9f5',
+              },
+              code: 'e5b855d51ad64fa0b6c180ed7e664c5d',
+              organization: {slug: 'sentry'},
+              status: 'installed',
+              uuid: '5379e8e1-0554-468f-90ca-2e0c88f1ac32',
+            },
+          ],
+        ],
+        [
+          `/organizations/${org.slug}/plugins/configs/`,
+          [
+            {
+              assets: [],
+              author: {name: 'Sentry Team', url: 'https://github.com/getsentry/sentry'},
+              canDisable: true,
+              contexts: [],
+              description: 'Forward Sentry events to Amazon SQS.',
+              doc: '',
+              featureDescriptions: [
+                {
+                  description: 'Forward Sentry errors and events to Amazon SQS.',
+                  featureGate: 'data-forwarding',
+                },
+              ],
+              features: ['data-forwarding'],
+              hasConfiguration: true,
+              id: 'amazon-sqs',
+              isHidden: false,
+              isTestable: false,
+              metadata: {},
+              name: 'Amazon SQS',
+              projectList: [],
+              resourceLinks: [
+                {
+                  title: 'Report Issue',
+                  url: 'https://github.com/getsentry/sentry/issues',
+                },
+                {
+                  title: 'View Source',
+                  url:
+                    'https://github.com/getsentry/sentry/tree/master/src/sentry_plugins',
+                },
+              ],
+              shortName: 'Amazon SQS',
+              slug: 'amazon-sqs',
+              status: 'beta',
+              type: 'data-forwarding',
+              version: '10.1.0.dev0',
+            },
+          ],
+        ],
+        [`/organizations/${org.slug}/repos/?status=unmigratable`, []],
+      ]);
+
+      const wrapper = mountWithTheme(
+        <IntegrationListDirectory params={{orgId: org.slug}} location={{search: ''}} />,
+        routerContext
+      );
+
+      expect(wrapper.find('SearchInput').exists()).toBeTruthy();
+      expect(wrapper.find('PanelBody').exists()).toBeTruthy();
+      expect(wrapper.find('IntegrationRow')).toHaveLength(5);
+      expect(
+        wrapper
+          .find('IntegrationRow')
+          .first()
+          .props().slug
+      ).toEqual('bitbucket');
```

Comment:
@NisanthanNanthakumar We should also have a test for the legacy plugin functionality. So the Github plugin should only show up if it's installed and it should have `(Legacy)` in the name.